libs/ui/primitives/src/button.tsx

```diff
@@ -36,12 +36,12 @@ export interface ButtonProps
   asChild?: boolean
 }
 
-const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+const Main = React.forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
**question:** This was done by Shadcn automatically?

Reply:
Shadcn called it Button, I changed the name to Main so that it is well presented in the design system component.